godtools/s3_download/action.yaml

```
@@ -0,0 +1,26 @@
+name: 'Piwik PRO godtools s3 download'
```

Godtools is essentially an abstraction layer, and for this, I would go with something like this:
```suggestion
name: 'Piwik PRO godtools download'
```
Currently, we're using AWS S3, but in the future, it might be Azure Blob or any other storage solution. It's possible that it could also retrieve data from our secure storage or even storage that isn't S3-compatible.